### PR TITLE
change the airport icon

### PR DESCRIPTION
**Pull request by massimosiani**

### CODE REVIEW

File: src/libs/map-engine.ts [hunk 1]
```diff
@@ -269,28 +275,45 @@ export default class MapEngine {
...
-    marker.setPosition({lat: latitude, lng: longitude});
-    return marker;
+    markerData.marker.setPosition({lat: latitude, lng: longitude});
+    drawAirportMarker(markerData);
+    return markerData;
 };
 
-const createOrUpdateMarker = (icaoNumber: string, longitude: number, latitude: number, markerData: MarkerData) => {
+function drawAirportMarker(markerData: AirportMarkerData): void {
+  const innerColor = markerData.enabled ? '#933400' : '#7a7a7a';
```
Comment (simoexpo) [inline src/libs/map-engine.ts]: I'll use the light orange here `#FB8F2D`

File: src/libs/map-engine.ts [hunk 2]
```diff
@@ -269,28 +275,45 @@ export default class MapEngine {
...
+  const dimension = baseMarkerDimension + markerData.zoom * zoomFactor;
+  const svg = `<?xml version="1.0" encoding="UTF-8"?>
+  <svg xmlns="http://www.w3.org/2000/svg" version="1.1" id="IconsRepoEditor" viewBox="-36.97 -36.97 443.69 443.69" width="${dimension}px" height="${dimension}px">
+    <g>
+      <path fill="${innerColor}" stroke="#FB8F2D" stroke-width="0" d="M328.646,130.413h10.773l30.328-59.871l-23.332-27.688h-16.813V22.696h11.988c2.127,0,3.852-1.727,3.852-3.854 c0-2.127-1.725-3.852-3.852-3.852h-30.497c-2.128,0-3.853,1.725-3.853,3.852c0,2.127,1.725,3.854,3.853,3.854h11.988v20.158h-72.172 l-23.333,27.688l30.328,59.871h10.773v64.029h-97.016v-32.5H36.334v32.5H0v160.314h45.664v-79.982h57.666v79.982H360V194.442 h-31.354V130.413z M267.974,115.249l-14.521-29.543h28.15v29.543H267.974z M107.33,241.108H29.996v-23.332h77.334V241.108z M199.33,241.108h-77.334v-23.332h77.334V241.108z M291.33,241.108h-77.334v-23.332h77.334V241.108z M309.404,115.249h-21.482 V85.706h21.482V115.249z M315.723,85.706h28.152l-14.522,29.543h-13.631V85.706z"></path>
```
Comment (simoexpo) [inline src/libs/map-engine.ts]: I'll try to increase `stroke-width` to 12 and use the dark orange `#933400` in `stroke`

File: src/libs/map-engine.ts (hunk 1 above)
Comment (massimosiani) [reply src/libs/map-engine.ts]: I rather disagree here. A lighter orange tends to disappear when close to a cluster of darker bordered icons.

File: src/libs/map-engine.ts (hunk 2 above)
Comment (massimosiani) [reply src/libs/map-engine.ts]: You are suggesting to use the same colors as the planes, but in this case there would be no distinction then. Shouldn't we highlight airports in same way?

**Pull request merged**